code/index.html

```diff
@@ -1,30 +1,182 @@
+                            <li>Relaxing massage</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-container">
+                <h3 id="accordion-header-3" class="accordion-header">
+                    <img src="./img/acc-workout.jfif" alt="image workout">
+                    <span>Workout</span>
+                    <span class="plus">+</span>
+                </h3>
+                <div class="accordion-text-container">
+                    <div class="accordion-text">
+                        <ul>
+                            <li>Body pump</li>
+                            <li>Zumba</li>
+                            <li>Spinning</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-container">
+                <h3 id="accordion-header-4" class="accordion-header">
+                    <img src="./img/acc-hands.jfif" alt="image manicure">
+                    <span>Manicure</span>
+                    <span class="plus">+</span>
+                </h3>
+                <div class="accordion-text-container">
+                    <div class="accordion-text">
+                        <ul>
+                            <li>French manicure</li>
+                            <li>Gel manicure</li>
+                            <li>Acryl manicure</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+
+        <!-- Signup form -->
+
+        <section class="form">
+            <h2>Book your spa-days here</h2>
+            <form action="http://httpbin.org/anything" method="POST">
+
+                <legend>Choose number of nights</legend>
+                <label class="dates">* Check in:
+                    <input type="date" id="check-in" name="check-in" max="2021-08-30">
+                </label>
+                <label class="dates">
+                    * Check out:
+                    <input type="date" id="check-out" name="check-out" value="2020-09-65" min="2020-09-06"
+                    max="2021-08-31">
+                </label>
+                <legend>Choose your room</legend>
+                <label>
+                    Single room
+                    <input type="radio" name="roomType" id="single-room" value="single-room" checked>
+                </label>
+                <label>
+                    Dubble room
+                    <input type="radio" name="roomType" id="double-room" value="dubble-room">
+                </label>
+                <label>
+                    Family room
+                    <input type="radio" name="roomType" id="family-room" value="family-room">
+                </label>
+                <label>
+                    Number of adults
+                    <input type="number" id="adults" name="adults" required>
```

Comment:
Great job using some simple validation for your form and also checked the room type 🤩